docs/securing-control-plane.md

```diff
@@ -33,20 +33,23 @@ for client authentication.
 By default, antrea-controller generates a self-signed certificate. You can
 override the behavior by [providing your own certificates](#providing-your-own-certificates).
 Either way, the antrea-controller will distribute the CA certificate as a
-ConfigMap named `antrea-ca` in the `kube-system` Namespace and inject it into
-the APIServices resources created by Antrea in order to allow its clients (i.e.
-antrea-agent, kube-apiserver) to perform authentication.
+ConfigMap named `antrea-ca` in the antrea-controller's Namespace and inject it
```

Comment:
should be mention that agent & controller should be running in the same Namespace? Or simply say "Antrea's Namespace" instead of "antrea-controller's Namespace"?

Reply:
Changed to Antrea's Namespace